### DISCUSSION TITLE
Issue/Project Board Integrations

### DISCUSSION THREAD

**Discussion by karkir0003** [community/community#34829] [Projects and Issues] [answered]
Are there any ongoing developments by the Github team on supporting integration of project management tools like Trello, Kanban, etc into Github Projects? For example, if a team uses Github for code version control storage, but the project management stuff lives on Trello, Kanban, and the like, it would probably make life easy if Github has a function within the Projects tab that says "Integrate project board" and you can enter a URL and anytime you make a change on your normal project management system, Github project board would sync that update to its project board. I would love an integration like this as a user.

**Comment by jfullstackdev** [comment, marked as answer]
there is an external service for that, but I don't know how much it is but it is a legit one 

https://zapier.com/apps/github/integrations/trello

**Reply by karkir0003** [reply]
Cool. thanks for pointing this out. This will be super helpful
